### PR TITLE
Run chart testing always with image from pr

### PR DESCRIPTION
**Pull request by mszostok**
**Description**

Changes proposed in this pull request:

- Run chart testing always with image from pr

**Related issue(s)**

- kyma-project/kyma#6644

### CODE REVIEW

**Comment by mszostok** [comment]
this pr need to be merge before: https://github.com/kyma-project/test-infra/pull/1927

File: hack/ci/run-chart-test.sh [hunk 1]
```diff
@@ -44,8 +44,20 @@ chart::lint() {
 }
 
 chart::install_and_test() {
+    pushd "${LOCAL_REPO_ROOT_DIR}"
```
Comment (polskikiel) [inline hack/ci/run-chart-test.sh]: Do we need these `pushd` and `popd` commands?

File: before-commit.sh [hunk 2]
```diff
@@ -14,44 +14,9 @@ echo "GOPATH:" + $GOPATH
...
-
-echo "? compile chart tests"
-${buildEnv} go test -v -c -o hb_chart_test ./test/charts/helm_broker_test.go
-goBuildResult=$?
-if [[ ${goBuildResult} != 0 ]]; then
-    echo -e "${RED}✗ go test -c ./test/charts/helm_broker_test.go ${NC}\n$goBuildResult${NC}"
-    exit 1
-else echo -e "${GREEN}√ go test -c ./test/charts/helm_broker_test.go ${NC}"
-fi
+source ./hack/build-binaries.sh
```
Comment (polskikiel) [inline before-commit.sh]: `make build-binaries`?

Comment (mszostok) [reply before-commit.sh]: I do not want to execute make in make

Currently the `before-commit.sh` is already execute in makefile, so executing `make build` will execute before-commit.sh and before-commit.sh will execute `make build-binaries` and that will be quite hard to follow, so I decide to just use the script:)

File: hack/ci/run-chart-test.sh (hunk 1 above)
Comment (mszostok) [reply hack/ci/run-chart-test.sh]: yes, because we are using the `make` commands

**Comment by mszostok** [comment]
https://github.com/kyma-project/test-infra/pull/1927 merged, so pipelines should now pass

/retest

File: charts/helm-broker/ci/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+## Providing Custom Test Values
```
Comment (klaudiagrz) [inline charts/helm-broker/ci/README.md]:
```suggestion
## Providing custom test values
```

File: charts/helm-broker/ci/README.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+
+In order to enable custom test values, add any number of *-values.yaml files to this directory. Only files with a suffix -values.yaml are considered. Instead of using the defaults, the chart is then installed and tested separately for each of these files using the --values flag.
+
+Please note that in order to test using the default values when using the ci directory, an empty values file must be present in the directory.
```
Comment (klaudiagrz) [inline charts/helm-broker/ci/README.md]:
```suggestion
If you want to perform testing using the default values, an empty `values.yaml` file must be present in the `ci` directory.
```

**Comment by mszostok** [comment]
Here is the issue: https://github.com/kyma-project/helm-broker/issues/84 :)
